databases/backends/postgres.py

```diff
@@ -66,7 +67,7 @@ async def connect(self) -> None:
 
     async def disconnect(self) -> None:
         assert self._pool is not None, "DatabaseBackend is not running"
-        await self._pool.close()
+        await asyncio.wait_for(self._pool.close(), timeout=30)
```

Comment:
As @tomchristie wrote, it is a good idea to make the timeout configurable, otherwise, 30 seconds can be too much for everybody.